BSP/Simulator_V2/Src/Simulator.c

```diff
@@ -19,6 +19,8 @@
 #include <stdbool.h>
 #include "cJSON.h" // for json parser
 
+#include "BSP_CAN.h" // CAN testing
```

Comment:
Is this required here?

Reply:
When I was doing testing directly in Sim init yes. I'll remove it once I figure out how to do a proper test file for the CAN.